src/main/java/seedu/address/commons/core/Messages.java

```diff
@@ -9,6 +9,10 @@
     public static final String MESSAGE_UNKNOWN_TYPE = "Unknown entity type";
     public static final String MESSAGE_INCOMPLETE_INPUT = "User input is incomplete";
     public static final String MESSAGE_INVALID_COMMAND_FORMAT = "Invalid command format! \n%1$s";
+    public static final String MISSING_TIEBREAK_METHODS = "Tie-break methods must be specified if "
+            + "tie-break prefix \"tb/\" is typed.";
+    public static final String MESSAGE_INVALID_TIE_BREAK = "The tie-break method specified does not exist or is not "
```

Comment:
Maybe specify the types of valid tiebreakers? Kind of like how the error message for project type will specify the valid enums to use. :bowtie: